custom_components/ai_thermostat/climate.py

```diff
@@ -64,7 +64,9 @@
 
 ATTR_STATE_WINDOW_OPEN = "window_open"
 ATTR_STATE_NIGHT_MODE = "night_mode"
-ATTR_STATE_CALL_FOR_HEAT = "summer"
+ATTR_STATE_SUMMER = "summer"
```

Comment:
The naming "Summer" is missleading.

We don't change the thermostat's operation based on the season, but on the forcast/outside temperature.

This is "call for heat" in the terminology of HVACs. So it should be named accordingly.

Compromise for the UI would be to use a ☀ (sun) or 🌤️ symbol instead "no call for heat" if the outside temperature is too high (or the forecast is too hot)